<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondWindow</class>
 <widget class="QDialog" name="SecondWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>140</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Формула 3000</string>
  </property>
  <widget class="QLabel" name="label_formula">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid #ba7f7f;
	color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_formula">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
background-color: rgb(255, 255, 255);
border:1px solid #ba7f7f;
	color: rgb(0, 0, 0);
 }



QSpinBox::up-button {
     subcontrol-origin: border;
     subcontrol-position: top right; /* позиция - в верхнем правом углу */
     width: 15px;
     height: 8px;
	 
	border-image: url(:/new/prefix1/png/up11.png);
     border-width: 1px;
 }
QSpinBox::up-button:hover { 
	border-image: url(:/new/prefix1/png/up12.png);
 }
QSpinBox::up-button:pressed {
	border-image: url(:/new/prefix1/png/up13.png);
 }



QSpinBox::down-button {
     subcontrol-origin: border;
     subcontrol-position: bottom right; /* позиция - в нижнем правом углу */
     width: 15px;
     height: 8px;
	 
	border-image: url(:/new/prefix1/png/down11.png);
     border-width: 1px;
 }
QSpinBox::down-button:hover { 
	 
	border-image: url(:/new/prefix1/png/down12.png);
 }
QSpinBox::down-button:pressed {
	
	border-image: url(:/new/prefix1/png/down13.png);
 }</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid #ba7f7f;
	color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Номер операции комбинаторики</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 250, 250);
color: rgb(0, 0, 0);
border:1px solid grey;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
